test/qfactors.jl

```diff
@@ -0,0 +1,30 @@
+@testset "qfactors" begin
+    # Check a Q factor gives the right RV precision
+    @test abs(Korg.precRV_fromQ(1100,100,8000)-30.470741601352298) < 1e-10
+
+    # Check that a solar spectrum at APOGEE resolution gives the expected Q factor
+    delLog = 6e-6; 
+    wavetarg = 10 .^range((start=4.179-125*delLog),step=delLog,length=8575+125)
+    minw, maxw = extrema(wavetarg);
+    
+    wl_lo, wl_hi, wl_step = 15_000, 17_000, 1//100
```

Comment:
Is there a particular reason to use a `Rational` here?

Reply:
Yes. Because for chi2 surface searching and interpolation, float precision issues are realy really annoying and have caused problem for apMADGICS before. So I want to make sure you are explicitly testing with rationals more than you are. I am taking this moment and my liberty in these tests to incept that.